packages/block-library/src/table/state.js

```diff
@@ -310,7 +301,7 @@ export function toggleSection( state, sectionName ) {
 	}
 
 	// Get the length of the first row of the body to use when creating the header.
-	const columnCount = get( state, [ 'body', 0, 'cells', 'length' ], 1 );
+	const columnCount = state.body?.[ 0 ]?.cells?.length ?? 1;
```

Comment:
`body` should always exist as per my manual checks, but I prefer not taking chances.